packages/ui/src/machines/authenticator/index.ts

```diff
@@ -25,47 +25,58 @@ export function createAuthenticatorMachine() {
       context: {
         user: undefined,
         config: {},
-        services: {},
+        services: { ...defaultServices },
         actorRef: undefined,
       },
       states: {
         // See: https://xstate.js.org/docs/guides/communication.html#invoking-promises
         idle: {
```

Comment:
`idle` state now starts with `Auth.currentAuthenticatedUser` call.

Knowing that, `idle` is a bad name (it's on me!) and should be named more explicitly. I scoped this out of this PR, because that would change our `route` names and I would rather deal with it in 3.0 changes.